hw8_Zhurylo/js/index.js

```diff
@@ -0,0 +1,80 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+    //CANCEL
+    let uhOhResult = `It is a pity that you did not want to enter your`
+
+
+    //AGE
+    //variables
+    const userYearOfBirth = Number(prompt('Enter your year of birth'))
+    let ageResult
+    const currentYear = new Date().getFullYear()
+    //conditions
+    if (!userYearOfBirth && !isNaN(userYearOfBirth)) {
```

Comment:
why do you need `!isNaN(userYearOfBirth)` here?

Reply:
because otherwise it didn't work the way I needed, and the result of the check didn't suit me.